vm/systemSmartContracts/auction.go

```diff
@@ -452,11 +500,13 @@ func (s *stakingAuctionSC) registerBLSKeys(
 func (s *stakingAuctionSC) updateStakeValue(registrationData *AuctionData, caller []byte) vmcommon.ReturnCode {
 	if len(registrationData.BlsPubKeys) == 0 {
 		log.Debug("not enough arguments to process stake function")
```

Comment:
maybe this message should be about no bls keys has been provided, or use the same pattern with expected and got like above

Reply:
refactored